data/brands/shop/car_repair.json

```diff
@@ -24,7 +24,7 @@
       "displayName": "A.T.U",
       "id": "atu-6db7c0",
       "locationSet": {"include": ["at", "de"]},
-      "matchNames": ["auto-teile-unger"],
+      "matchNames": ["auto-teile-unger", "ATU", "A.T.U."],
```

Comment:
Names are matched with the punctuation removed, so "ATU" and "A.T.U." will be considered redundant.  It's ok to add them here, but the build script will remove them automatically because they don't do anything.